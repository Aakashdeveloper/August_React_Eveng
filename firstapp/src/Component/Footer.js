import React from 'react';

const Footer = (props) => {
    return(
        <React.Fragment>
            <hr/>
            <center>
                <h3>Developed with &hearts; by DeveloperFunnel  &copy;
                    {props.month} {props.year}  
                </h3>
            </center>
        </React.Fragment>
    )
}

export default Footer;