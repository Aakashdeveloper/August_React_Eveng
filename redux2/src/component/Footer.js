import React from 'react';

const Footer = () => {
    return(
        <header>
            <div>
               <p>&copy; DeveloperFunnel</p>
            </div>
        </header>
    )
}

export default Footer;