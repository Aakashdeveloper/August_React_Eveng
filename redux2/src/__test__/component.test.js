import React from 'react';
import Footer from '../component/Footer';
import {create} from 'react-test-renderer';

describe('Snapshot test',() => {
    test('Testing footer',() => {
        let tree = create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})