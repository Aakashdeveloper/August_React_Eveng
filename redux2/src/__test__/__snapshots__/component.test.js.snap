// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test Testing footer 1`] = `
<header>
  <div>
    <p>
      Â© DeveloperFunnel
    </p>
  </div>
</header>
`;
